src/ZipkinOpenTracing/SpanContext.php

```diff
@@ -37,15 +37,15 @@ public function getIterator()
     /**
      * @inheritdoc
      */
-    public function getBaggageItem($key)
+    public function getBaggageItem(string $key): ?string
     {
         return \array_key_exists($key, $this->baggageItems) ? $this->baggageItems[$key] : null;
     }
 
     /**
      * @inheritdoc
      */
-    public function withBaggageItem($key, $value)
+    public function withBaggageItem(string $key, string $value): ?OTSpanContext
```

Comment:
I think there is a mistake here, it should be `OTSpanContext` instead of `?OTSpanContext` but the error comes from the library itself https://github.com/opentracing/opentracing-php/blob/master/src/OpenTracing/SpanContext.php#L36. I am fixing it in there.

Reply:
I dealt with this just now.

Take a look again.

Thanks.